src/components/sendV2/summary/index.js

```diff
@@ -1,19 +1,21 @@
 /* istanbul ignore file */
 import { connect } from 'react-redux';
 import { translate } from 'react-i18next';
-import { sent } from '../../../actions/transactions';
+import { transactionCreated, resetTransactionResult } from '../../../actions/transactions';
 import { tokenMap } from '../../../constants/tokens';
 import Summary from './summary';
 
 const mapStateToProps = state => ({
   account: state.account,
   failedTransactions: state.transactions.failed,
   pendingTransactions: state.transactions.pending,
+  transactions: state.transactions,
```

Comment:
Isn't possible to remove upper lines, since you already have that data into the `transactions`??